migrations/20170605161512_rename-followers.js

```diff
@@ -1,11 +1,13 @@
 exports.up = function (knex, Promise) {
   return knex.schema.table('communities_tags', table => {
-    table.integer('followers').defaultTo(0)
+    table.dropColumn('followers')
```

Comment:
pretty sure this migration will fail on production because `followers` doesn't exist